availability_zones.yaml

```
@@ -1,22 +1,24 @@
-
-online:
-  - online
-EU:
-  - online
-  - EU
-NERSC:
-  - online
-  - NERSC
-NCAR:
-  - online
-  - NCAR
-UK:
-  - online
-  - UK
-JAPAN:
-  - online
-  - JAPAN
-
-CN:
-  - online
-  - CN
\ No newline at end of file
+
+online:
+  - online
+EU:
+  - online
+  - EU
+NERSC:
+  - online
+  - NERSC
+NCAR:
+  - online
+  - NCAR
+UK:
+  - online
+  - UK
+JAPAN:
+  - online
+  - JAPAN
+
+CN:
+  - online
+  - CN
+SP:
+  - IAG/USP
```

I'm assuming you want to have the data available locally at `SP` to be included in your catalog, and additionally want to access data available `online`. I further assume that you'd want to prefer accessing locally available data if possible. If this is the case, the right configuration would be:

```suggestion
SP:
  - online
  - SP
```